targets/unity-v2/source/Client/Authentication/Strategies/EmailPasswordAuthStrategy.cs

```diff
@@ -26,8 +26,42 @@ public void Authenticate(PlayFabAuthService authService, Action<LoginResult> res
             {
                 Email = authService.Email,
                 Password = authService.Password,
-                InfoRequestParameters = authService.InfoRequestParams
-            }, resultCallback, errorCallback);
+                InfoRequestParameters = authService.InfoRequestParams,
+                TitleId = PlayFabSettings.TitleId
+            }, resultCallback, error =>
```

Comment:
nitpick, can the lambda start on its own line (with parentheses around the parameter?)